database/generateProducts.js

```diff
@@ -0,0 +1,55 @@
+const casual = require('casual');
+// const { Client } = require('pg');
+
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+
+const writer = csvWriter({
+  separator: ';',
```

Comment:
is this making your csv be semi colon separated instead of comma separated?

Reply:
yup, because i have commas inside text im generated for certain columns